### PR TITLE
[13.0][ADD] website_sale_cart_expire

### PR DESCRIPTION
**Pull request by ivantodorovich**
Module that allows to automatically cancel carts without activity after a configurable time.

### CODE REVIEW

File: website_sale_cart_expire/tests/test_website_sale_cart_expire.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        self.order_1.website_id = self.website_1
+        self.assertFalse(self.order_1.cart_expire_date)
+
+    def test_expire_scheduler(self):
+        # Case 1: We haven't reached the expire date yet
+        self.env["website"]._scheduler_website_expire_cart()
+        for order in self.orders:
+            self.assertEqual(order.state, "draft")
+        # Case 2: We have reached website 2 expire date
+        with mock.patch("odoo.fields.Datetime.now") as mock_now:
```
Comment (chienandalu) [inline website_sale_cart_expire/tests/test_website_sale_cart_expire.py:44]: Do you now freezegun? It's been made available through a lot of OCA repositories (including this one) and it's very cool to mock dates in tests

**Review by chienandalu — APPROVED** [review]
Cool :+1:

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by ioans73 — APPROVED** [review]
LGTM

**Pull request merged**